Makefile

```diff
@@ -106,10 +111,56 @@ CARGO_FLAGS += --release
 TARGET_DIR=$(BINDIR)/target/release
 endif
 
+ifeq ($(COV),1)
```

Comment:
We should add comments as to why. There's no way anyone remembers this in the future.